### DISCUSSION TITLE
[Enhancement] MAUI needs to have some way to handle images such as FFImageLoading

### DISCUSSION THREAD

**Discussion by rs-mobitech** [dotnet/maui#392] [Ideas]
## Summary

FFImageLoading is a great package but it's not so well documented and recently seems to have less and less support from the developers.  I believe many apps are using this and it's the only good way to handle many of our image needs. 

I would like to suggest that MAUI needs a way to handle all of our image requirements.

**Comment by Syed-RI** [comment]
Shocking that there is no movement here!

**Comment by hartez** [comment]
> I would like to suggest that MAUI needs a way to handle all of our image requirements.

Can you be more specific about which requirements?

**Reply by Syed-RI** [reply]
I would imagine that would be the following:

- Placeholder image
- Retry policy
- Simple image transformation (black and white, blur etc)
- Caching (already available)
- SVG/GIF support (already available)
- Relevant events (load error, success etc)
